qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -65,7 +66,7 @@ class SabreSwap(TransformationPass):
     `arXiv:1809.02573 <https://arxiv.org/pdf/1809.02573.pdf>`_
     """
 
-    def __init__(self, coupling_map, heuristic="basic", seed=None, fake_run=False):
+    def __init__(self, coupling_map, heuristic="basic", seed=None, fake_run=False, properties=None):
```

Comment:
It would be good not to rely on `BackendProperties` with the swap to `BackendV2` already in place - for new stuff, we should probably try and use either this or `Target` as appropriate.